build/parseChangelog.c

```diff
@@ -30,10 +30,12 @@ static int sameDate(const struct tm *ot, const struct tm *nt)
 	    ot->tm_wday == nt->tm_wday);
 }
 
+#define TZ_MAX_LENGTH 80
```

Comment:
It would be great if this number was described. I mean why not 67 or 93 :)

Reply:
Just use dynamic allocation to avoid arbitrary limits that need explanations :)

Reply:
Thank you. In the new version dynamic allocation is used.